src/advanced/bitcoin-anchoring.md

```diff
@@ -346,7 +305,7 @@ equal to `/api/services/btc_anchoring`.
 ### Actual Address
```

Comment:
Do you plan to describe private endpoints?

Reply:
No, I don't think because this is an internal implementation details and the end user should not know about such details.